### ISSUE TITLE
add support for tmi.twitch.tv

### ISSUE THREAD

**Issue by enilsen16** [twitch-irc/twitch-irc-api#1]
The tmi.twitch endpoint seems really poorly documented and only somewhat reliable, but can provide useful information.

for instance:
```
https://tmi.twitch.tv/group/user/lirik/chatters
```

**Comment by Schmoopiie** [comment]
Yes, good idea.. I also want to add an alias for the .call method because it sounds bad. We could use .tmi for the internal api and something else for the official api.. I just can't think of a name right now :smile:
